src/FakeItEasy/Raise.cs

```diff
@@ -77,7 +77,9 @@ public static TEventHandler With<TEventHandler>(params object[] arguments)
         /// with all CLR languages (for example Visual Basic).
         /// To raise non-standard events from other languages, use <see cref="Raise.FreeForm{TEventHandler}"/>.
         /// </summary>
+#pragma warning disable CA1034

```

Comment:
I've always enjoyed the justifications we've had on the suppressed warnings, or failing that, an indication of what we're suppressing, not unlike what you've done in FakeManager.cs. Would you consider giving a bit of a hint for these pragmas?

Reply:
Sure, I'll add a comment

Reply:
Thanks.